bftengine/src/bftengine/ControlHandler.hpp

```diff
@@ -0,0 +1,42 @@
+// Concord
+//
+// Copyright (c) 2018-2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "Replica.hpp"
+#include <functional>
+
+namespace bftEngine {
+
+class ControlHandler : public IControlHandler {
```

Comment:
Please add documentation so that it becomes clear what the ControlHandler does. I saw that there is documentation for the IControlHandler class, maybe move it into this file?